pkg/report/handler.go

```diff
@@ -2,11 +2,19 @@ package report
 
 import (
 	"fmt"
+	"io/ioutil"
 	"net/http"
 
+	khttp "github.com/hellofresh/kangal/pkg/core/http"
+	kk8s "github.com/hellofresh/kangal/pkg/kubernetes"
+
 	"github.com/go-chi/chi"
+	"github.com/go-chi/render"
+	"k8s.io/apimachinery/pkg/api/errors"
 )
 
+var httpClient = &http.Client{}
```

Comment:
Please provide timeout for your HTTP client or handle timeouts via context like we are doing here:
https://github.com/hellofresh/kangal/blob/master/pkg/proxy/proxy.go#L265-L266

Reply:
If reports are big we might get a timeout error

Reply:
I added an 30 seconds timeout